website/content/docs/commands/job/restart.mdx

```diff
@@ -0,0 +1,206 @@
+---
+layout: docs
+page_title: 'Commands: job restart'
+description: |
+  The job restart command is used to restart allocations for a job.
+---
+
+# Command: job restart
+
+The `job restart` command is used to restart or reschedule allocations for a
+particular job.
+
+## Usage
+
+```plaintext
+nomad job restart [options] <job>
+```
+
+The `job restart` command requires a single argument, specifying the job ID to
+restart.
+
+The command can operate in batches and wait until all restarted or
+rescheduled allocations are running again before proceeding to the next batch.
+It is also possible to specify additional time to wait between batches.
+
+Allocations can be restarted in-place or rescheduled. When restarting
+in-place the command may target specific tasks in the allocations, restart
+only tasks that are currently running, or restart all tasks, even the ones
+that have already run. Allocations can also be targeted by groups and tasks.
+When both groups and tasks are defined only the tasks for the allocations of
+those groups are restarted.
```

Comment:
It seems worth specifically calling out that we're only checking client status here and not health checks.

Reply:
Added some content about it:

```
When rescheduling, the current allocations are stopped triggering the Nomad
scheduler to create replacement allocations that may be placed in different
clients. The command waits until the new allocations have client status `ready`
before proceeding with the next batch. Services health checks are not taken
into account.
```